Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -70,6 +74,21 @@ public interface IGitModule
         /// <summary>Indicates HEAD is not pointing to a branch (i.e. it is detached).</summary>
         bool IsDetachedHead();
 
+        /// <summary>
+        /// Convert the path for the Git executable.
+        /// For WSL Git, the path will be adjusted.
+        /// </summary>
+        /// <param name="path">Path as seen by the Git Extensions native (Windows) executable.</param>
+        /// <returns>Posix path if Windows Git, WSL path for WSL Git.</returns>
+        public string GetPathForGitExec(string? path);
+
+        /// <summary>
+        /// Convert a path to native (Windows) format.
+        /// </summary>
+        /// <param name="path">Path as seen by the Git executable, possibly WSL Git.</param>
+        /// <returns>Path in in Windows format with native file separators.</returns>
+        public string GetPathForNative(string path);
```

Comment:
Would it makes sense to name these as follows?
```cs
        public string GetDistroPath(string? path);
        public string GetWindowsPath(string? path);
```

Reply:
I still have a dream to support more than windows so I prefer the current name.

Reply:
When we get to support some other OS we can rename it, I promise :)

Reply:
I can change. Will also change GitExec names, to be consistent. (Most Native names/comments).

Will write something in the comments like:
`With Windows path the GitExtensions.exe native path format is meant, this should be renamed if GE is running native on other OSes.`

Reply:
Updated the naming to `GetWindowsPath` and `GetGitExecPath` (the last is more descriptive to me, the distro may use several Git executables).